_content/monographs/zen-texts_mcrea-tokiwa-yoshida-heine.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Zen Texts"
+translator: "	John R. McRae and Gishin Tokiwa and Osamu Yoshida and Steven Heine"
```

Comment:
When we have monographs like this (e.g. Bhikkhu Bodhi's Majjhimā Nikaya) I just put them as authors